config/crossSections.yaml

```diff
@@ -0,0 +1,748 @@
+  crossSec: 53870.0
+  reference: XSDB WJetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2021-06-21 10:00:29
+  unc: 129.7
+W1JetsToLNu_LO:
+  crossSec: 8927.0
+  reference: XSDB W1JetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2021-06-21 09:59:23
+  unc: 24.09
+W2JetsToLNu_LO:
+  crossSec: 2809.0
+  reference: XSDB W2JetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2021-06-21 09:59:25
+  unc: 8.201
+W3JetsToLNu_LO:
+  crossSec: 826.3
+  reference: XSDB W3JetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2021-06-21 09:59:27
+  unc: 2.51
+W4JetsToLNu_LO:
+  crossSec: 544.3
+  reference: XSDB W4JetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2018-07-10 12:51:40
+  unc: 1.656
+WJetsToLNu_HT-70To100_LO:
+  crossSec: 1264.0
+  reference: XSDB WJetsToLNu_HT-70To100_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2021-06-21 10:00:11
+  unc: 3.696
+WJetsToLNu_HT-100To200_LO:
+  crossSec: 1256.0
+  reference: XSDB WJetsToLNu_HT-100To200_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2021-06-21 09:59:59
+  unc: 3.724
+WJetsToLNu_HT-200To400_LO:
+  crossSec: 335.5
+  reference: XSDB WJetsToLNu_HT-200To400_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2021-06-21 10:00:04
+  unc: 1.007
+WJetsToLNu_HT-400To600_LO:
+  crossSec: 45.25
+  reference: XSDB WJetsToLNu_HT-400To600_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2021-06-21 10:00:06
+  unc: 0.136
+WJetsToLNu_HT-600To800_LO:
+  crossSec: 10.97
+  reference: XSDB WJetsToLNu_HT-600To800_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2021-06-21 10:00:09
+  unc: 0.03307
+WJetsToLNu_HT-800To1200_LO:
+  crossSec: 4.933
+  reference: XSDB WJetsToLNu_HT-800To1200_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2021-06-21 10:00:13
+  unc: 0.01492
+WJetsToLNu_HT-1200To2500_LO:
+  crossSec: 1.16
+  reference: XSDB  WJetsToLNu_HT-1200To2500_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2021-06-21 10:00:02
+  unc: 0.003605
+WJetsToLNu_HT-2500ToInf_LO:
+  crossSec: 0.008001
+  reference: XSDB  WJetsToLNu_HT-2500ToInf_TuneCP5_13TeV-madgraphMLM-pythia8 modifiedOn 2018-07-10 12:51:54
+  unc: 2.486e-05
+W_To_TauNu_M-200:
+  crossSec: 7.253
+  reference: XSDB WToTauNu_M-200_TuneCP5_13TeV-pythia8-tauola modifiedOn 2021-06-21 10:00:41
+  unc: 0.01233
+W_To_MuNu_M-200:
+  crossSec: 7.268
+  reference: XSDB WToMuNu_M-200_TuneCP5_13TeV-pythia8 modifiedOn 2021-06-21 10:00:37
+  unc: 0.01236
+ZZTo4L:
+  crossSec: 1.325
+  reference: XSDB ZZTo4L_TuneCP5_13TeV_powheg_pythia8 modifiedOn 2020-10-10 11:49:47
+  unc: 0.00122
+ZZTo2L2Nu:
+  crossSec: 0.9738
+  reference: XSDB ZZTo2L2Nu_TuneCP5_13TeV_powheg_pythia8 modifiedOn 2021-09-28 13:26:27 mll>4GeV
+  unc: 0.0009971
+ZZTo2Q2L:
+  crossSec:  3.693e # 5.52
+  reference: ttH AN-19-111
+  unc: 4.149e-03
+ZZTo2Q2Nu:
+  crossSec: 4.561
+  reference: XSDB ZZTo2Q2Nu_TuneCP5_13TeV modifiedOn 2020-09-25 05:03:18
+  unc: 0.01533
+ZZTo4Q:
+  crossSec:  3.302 #6.912 <-- previous value. The new one has been evaluated on many MiniAOD
+  reference: unknown
+  unc: 6.328e-03
```

Comment:
- take from XSDB or recompute from miniAOD. For recomputed values as reference put "from GenXSecAnalyzer"
- 3.693e -> 3.693
- ZZTo4Q is smaller than ZZTo2Q2L -> looks suspicious.

Reply:
But ZZTo2L2Q has a cut on mll (as it seems from the dataset) 
```/ZZTo2Q2L_mllmin4p0_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIISummer20UL18MiniAODv2-106X_upgrade2018_realistic_v16_L1v1-v2/MINIAODSIM```

Reply:
I see. Please, specify it in the name of cross section to avoid confusions. E.g. for samples with mass cut add `_m(ll)>4GeV`

Reply:
ok